jsconfig.json

```diff
@@ -5,7 +5,7 @@
     "checkJs": true,
     "esModuleInterop": true,
     "moduleResolution": "node",
-    "module": "ES2020",
+    "module": "es2022",
```

Comment:
not an issue, just curious – what's introduced that requires this to be es2022?

Reply:
Dynamic imports with top level await.

```
const { authConfig } = await import('./components/auth.js');
```

That would have been flagged without es2022.